tasks/ssh.yml

```diff
@@ -6,21 +6,21 @@
     line: "{{ item.line }}"
     state: present
   with_items:
-    - regexp: "^PasswordAuthentication"
+    - regexp: "^?PasswordAuthentication"
```

Comment:
why did you just add a ? doesn't that mean it can be present anywhere in the line?